libV2/client/index.js

```diff
@@ -83,8 +83,10 @@ class UtapiClient {
             accessKeyId: config && config.accessKeyId,
             secretAccessKey: config && config.secretAccessKey,
         };
-        assert.notStrictEqual(this._credentials.accessKeyId, undefined, 'you must provide an accessKeyId');
-        assert.notStrictEqual(this._credentials.secretAccessKey, undefined, 'you must provide a secretAccessKey');
+
+        if (!this._credentials.accessKeyId || !this._credentials.secretAccessKey) {
+            this._logger.warn('Now access or secret key provided, client limited to push only');
```

Comment:
```suggestion
            this._logger.warn('No access or secret key provided, client limited to push only');
```

Reply:
I just realized I typoed the commit msg about a typo fix.

Reply:
haha